pages/mentoring_faqs.md

```diff
@@ -16,7 +16,7 @@ In the meantime, here are some recommended resources:
 
 - [The Science and Art of Receiving Feedback Well](https://www.amazon.com/Thanks-Feedback-Science-Receiving-Well-ebook/dp/B00DMCV0XE/)
 - [Unlearning Toxic Behaviours in a Code Review Culture - Conference talk by Sandya Sankarram](https://drive.google.com/file/d/1ROZA9yKpEa8suiqVXtPO0IJtqa95SOjE/view) and related [Blog Post](https://medium.freecodecamp.org/unlearning-toxic-behaviors-in-a-code-review-culture-b7c295452a3c)
-- [How Positive and Negative Feedback Motivate Goal Pursuit](https://faculty.chicagobooth.edu/ayelet.fishbach/research/FEF%20Compass%202010.pdf)
+- How Positive and Negative Feedback Motivate Goal Pursuit
```

Comment:
```suggestion
- How Positive and Negative Feedback Motivate Goal Pursuit
[How Positive and Negative Feedback Motivate Goal Pursuit](https://web.archive.org/web/20181227202107/https://faculty.chicagobooth.edu/ayelet.fishbach/research/FEF%20Compass%202010.pdf)
```
Without the resource, perhaps the line doesn't make sense.  This gives the way back machine link to that document so that it is still available.

Reply:
Yes, you are correct. But I wasn't able to find another link for the source.

Reply:
That's OK.  Feel free to update if you like the change.

Reply:
Sure, thank you

Reply:
I am new to this pull request and stuff. Please let me know if I have done your suggestion correctly,